recipe/meta.yaml

```diff
@@ -10,25 +11,29 @@ source:
   sha256: 7ef2564f656ec48c4621f555c4431c4ec71900f3c45c8d2203f9c78446e8391c
 
 build:
-  number: 0
+  number: 1
+  noarch: python
   script: {{ PYTHON }} -m pip install . -vv
-  skip: true  # [not linux]
 
 requirements:
   host:
     - pip
-    - python
+    - python >=3.0
```

Comment:
From https://github.com/bheinzerling/bpemb/blob/7a6f996e0e03072738d694606a6cac2991bcb77d/setup.py#L27 There is no real minimum version that I could find.

Reply:
Made it python3 only with https://github.com/conda-forge/bpemb-feedstock/pull/2/commits/e5c568b92d48e859e7c20929de8d5f4dc855c691. Let me know if I should add a different minimum version.